src/limbo/bayes_opt/boptimizer.hpp

```diff
@@ -30,7 +30,8 @@ namespace limbo {
             boost::parameter::optional<tag::initfun>,
             boost::parameter::optional<tag::acquifun>,
             boost::parameter::optional<tag::stopcrit>,
-            boost::parameter::optional<tag::modelfun>> boptimizer_signature;
+            boost::parameter::optional<tag::modelfun>>
+            boptimizer_signature;
```

Comment:
In the style file, there doesn't seem to be any such limit. My guess is that there is a default setting to limit the length of lines as much as possible.

Does it bother you ?